modules/i3s/src/lib/tileset/i3s-tile-manager.js

```diff
@@ -0,0 +1,37 @@
+const STATUS = {
```

Comment:
Nit:
- The existing `3DTiles` implementation stores tile content loading status on the `Tile3D` class. 
- This PR adds a separate manager class for metadata loading status.
Longer term, I wonder if it wouldn't be more maintainable to have one mechanism for all cases?